lib/glific/communications/message.ex

```diff
@@ -245,7 +248,7 @@ defmodule Glific.Communications.Message do
   defp receive_text(message_params) do
     message_params
     |> Messages.create_message()
-    |> publish_data(:received_message)
+    |> publish_data(get_received_msg_publish_event(message_params))
```

Comment:
get value before passing into the function. looks cleaner

message_event = get_received_msg_publish_event(message_params))
message_params
    |> Messages.create_message()
    |> publish_data(message_event)